soundfile.py

```diff
@@ -968,7 +979,7 @@ def buffer_read_into(self, buffer, dtype):
         frames = self._cdata_io('read', cdata, ctype, frames)
         return frames
 
-    def write(self, data):
+    def write(self, data, compression_level=None, bitrate_mode=None):
```

Comment:
This should probably be a parameter of the `SoundFile` object, not the write function, since you can't use different values in several successive write calls.